### ISSUE TITLE
Tabbing out of an Upload field skips the submit button

### ISSUE THREAD

**Issue by kjohnson** [wpninjas/ninja-forms-uploads#330]
On a simple contact form I can tab through and submit the form using only the keyboard.

However, if I come across an Upload filed (and keyboard through uploading a file), the tab key seems to skip over the submit button.

**Comment by polevaultweb** [comment]
@kjohnson i can't recreate. Once the file progress bar has finished, the next tab goes to the 'delete' for the uploaded file, and then the next tab lands on the submit button.

It happens on the customers site if there are validation errors.

**Comment by wpnzach** [comment]
@polevaultweb to clarify, you can recreate, but only if there are form validation errors?

**Comment by polevaultweb** [comment]
@wpnzach yes, but the behaviour I see is the tab to the top to show validation errors. The tabbing sequence of FU is correct so I think this might be a core thing. Eg. what would happen if you tab off the last text box to submit but have validation errors on a form without FU?
